go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a // indirect
 	github.com/mitchellh/mapstructure v1.2.2
 	github.com/openzipkin/zipkin-go v0.2.2 // indirect
+	github.com/pkg/errors v0.9.1
```

Comment:
Please do not introduce dependency. Use Go 1.13's error wrapping: https://blog.golang.org/go1.13-errors instead.